pkg/kubelet/cm/memorymanager/fake_memory_manager.go

```diff
@@ -76,19 +84,21 @@ func (m *fakeManager) State() state.Reader {
 
 // GetAllocatableMemory returns the amount of allocatable memory for each NUMA node
 func (m *fakeManager) GetAllocatableMemory() []state.Block {
-	klog.InfoS("Get Allocatable Memory")
+	m.logger.Info("Get Allocatable Memory")
 	return []state.Block{}
 }
 
 // GetMemory returns the memory allocated by a container from NUMA nodes
 func (m *fakeManager) GetMemory(podUID, containerName string) []state.Block {
-	klog.InfoS("Get Memory", "podUID", podUID, "containerName", containerName)
+	m.logger.Info("Get Memory", "podUID", podUID, "containerName", containerName)
 	return []state.Block{}
 }
 
 // NewFakeManager creates empty/fake memory manager
-func NewFakeManager() Manager {
+func NewFakeManager(ctx context.Context) Manager {
+	logger := klog.LoggerWithName(klog.FromContext(ctx), "memory-mgr.fake")
```

Comment:
I used "cpu.fake" for brevity. We should keep the naming consistent, but I didn't have made my mind yet. I don't have streong opinoins either, `exceptTheFactIWouldReallyLikeToAvoidLongIdentifiersWhichTakeALotOfHorizontalSpaceWhileProvidingQuestionableInformationBenefits`

Reply:
Will think more about this as well, Thank you for raising this!

Reply:
Long identifiers affecting readability is definitely a valid concern. IMO keeping naming consistent while maintaining brevity is a good balance to aim for. . Open to any suggestions in this area!